GEOparse/GEOTypes.py

```diff
@@ -621,7 +621,7 @@ def phenotype_data(self):
                     else:
                         tmp[key] = ",".join(value)
                 pheno_data[gsm_name] = tmp
-            self._phenotype_data = DataFrame(pheno_data).T
+            self._phenotype_data = DataFrame(pheno_data)
```

Comment:
This is the structure of the table where index is an experiment and descriptions are in columns. I think this should stay as it is (at least in python and pandas version I tested).